src/humanReadableSize/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+  <translation id="unit_size_B" qtlid="121975">o</translation>
```

Comment:
It should be prefix :) like `ua_humanReadbleSize_unit_size_B`, no?

Reply:
Why `ua` ?